### ISSUE TITLE
Indent rule: missing option to enforce indentation of break

### ISSUE THREAD

**Issue by lo1tuma** [eslint/eslint#1798]
The indent rule recognizes if a `break` within of a switch statement should be indented or not.
The following two examples are both valid:

```js
switch (a) {
case '1':
    b();
break;
default:
    c();
break;
}
```

```js
switch (a) {
case '1':
    b();
    break;
default:
    c();
    break;
}
```

This one fails:
```js
switch (a) {
case '1':
    b();
break;
default:
    c();
    break;
}
```
```sh
test.js
  7:0  error  Expected indentation of 0 characters  indent
```

What I want is an option to enforce a consistent indentation of `break`.

**Comment by gyandeeps** [comment]
As per the rewrite if the `indent` rule, `break` statements get indented to the same level as the content body of the case statement by default.
As per this issue, do we need something different or is it ok to close this issue.

**Comment by lo1tuma** [comment]
For me it be ok to close this issue. I don't think  if I've ever seen code where break has the same indentation as the case label.

**Comment by gyandeeps** [comment]
closing based on the above discussion.
